### PR TITLE
Metadata.md: Document that contiguousness is a must

### PR DESCRIPTION
**Pull request by Olf0**
… as described at https://github.com/sailfishos-chum/sailfishos-chum-gui/issues/111#issuecomment-1046241434

### CODE REVIEW

**Comment by Olf0** [comment]
Side note: The [RPM packaging guide](https://rpm-packaging-guide.github.io/#what-is-a-spec-file) calls each part, which is introduced by a `%foo` directive a *section*, while all the `Bar:` directives, which are followed by a *field* containing a value, are part of (and make up) the "preamble section".

File: Metadata.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
 # Additional metadata for SailfishOS:Chum
 
-The basic specification for the additional metadata is that of AppStream.  While it is a goal to leverage AppStream metadata directly for SailfishOS:Chum packages and the SailfishOS:Chum GUI application, we are not at that stage yet, and are instead using a method of embedding metadata in the "Description" field of the rpm spec file.  An example of how this is done follows:
+The basic specification for the additional metadata is that of AppStream.
+While it is a goal to leverage AppStream metadata directly for SailfishOS:Chum packages and the SailfishOS:Chum GUI application, we are not at that stage yet, and are instead using a method of embedding metadata in the `%description` section of the RPM spec file.
+
+Note that it is important, that the part from the `%if "%{?vendor}" == "chum"` to the `%endif` is the last and a contiguous paragraph of the `%description` section, i.e., it must not contain empty or comment lines (or any other line, which is evaluated to an empty line).
```
Comment (rinigus) [inline Metadata.md:6]: In principle, you don't have to include it within `%if` / structure. Maybe that should be stated as well.

Comment (Olf0) [reply Metadata.md:6]: Done by [commit `10921ef`](https://github.com/sailfishos-chum/main/pull/70/commits/10921ef18391893df6df241f619ef8c663a4e624).

**Comment by rinigus** [comment]
LGTM

**Pull request merged**